docs/pages/project/contributing/contributing-docs.md

```diff
@@ -147,6 +147,25 @@ Meshery documentation is made of these components:
 
   _Note: From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`. There are two Jekyll configuration, `jekyll serve` for developing locally and `jekyll build` when you need to generate the site artifacts for production._
 
+#### Note
+While performing the above step, if you're facing errors with message like below...
+
+`Your ruby version is x.x.x but your Gemfile specified 2.7.x`
+
+This is because Jekyll always consider the exact version of Ruby unlike JS
+
+So, you need to follow either of the two steps to resolve this problem
+  - Install the required ruby version by using `rvm` or by any means given above
+  - If you can't unable to install the required version, then manually configure the `Gemfile` as below
+  ```
+  source "https://rubygems.org" 
+  ruby '2.7.1' //to any version you have installed
+  ```
+  Automatically the `Gemfile.lock` will update once the `make site` is given
```

Comment:
Mainly in windows the `make site` creates a lot of problem. So I think we can also mention that we can also run `bundle exec jekyll serve` alongside the `make site` command which also does the same thing and run without any error in windows.

Reply:
Made the changes, let's see if it needs precise changes too...